scripts/seed_sample_data.py

```
@@ -0,0 +1,254 @@
+                    "max_depth": 10,
+                    "random_state": 42
+                },
+                "training_samples": 1500,
+                "validation_accuracy": 0.873
+            },
+            accuracy_score=87.3
+        )
+        db.add(predictive_model)
+        
+        db.commit()
+        print(f"✅ Successfully seeded:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)